docs/reference/environment-variables.mdx

```diff
@@ -90,10 +90,10 @@ redirects user to the HTTPS version of a page in this case. Use the
   invalidate parts of Front-Commerce cache. See
   [Invalidating the cache](/docs/advanced/graphql/dataloaders-and-cache-invalidation#invalidating-the-cache)
   for more details.
-- `FRONT_COMMERCE_CART_CACHE_ENABLE`: allows to enable [the cart cache (only
-    available when running Front-Commerce with Magento2)](/docs/magento2/advanced#caching-customers-cart)
-- `FRONT_COMMERCE_CURRENT_CUSTOMER_CACHE_ENABLE`: allows to enable [the current customer cache (only
-    available when running Front-Commerce with Magento2)](/docs/magento2/advanced#caching-current-customers-information)
+- `FRONT_COMMERCE_CART_CACHE_ENABLE`: allows to enable
+  [the cart cache (only available when running Front-Commerce with Magento2)](/docs/magento2/advanced#caching-customers-cart)
+- `FRONT_COMMERCE_CURRENT_CUSTOMER_CACHE_ENABLE`: allows to enable
+  [the current customer cache (only available when running Front-Commerce with Magento2)](/docs/magento2/advanced#caching-current-customers-information)
```

Comment:
This seems like formatting changes? is it an IDE config issue or was this never formatted correctly?

Reply:
It makes it hard to review

Reply:
Given the other changes, my guess is it was never formatted correctly (my IDE didn't change one bit for the past months). I'll git `add -p`

The only changes from this MR are the ones under `# Attraqt`

Reply:
Done, should be easier now.